src/types/timestamp.rs

```diff
@@ -1431,4 +1431,38 @@ mod ionc_tests {
         assert_eq!(source, converted_source);
         Ok(())
     }
+
+    #[test]
+    fn ion_eq_fraction_seconds_mixed_mantissa() {
+        let t1 = Timestamp {
+            date_time: NaiveDateTime::from_str("1857-05-29T19:25:59.100").unwrap(),
```

Comment:
What is this date? Arbitrary? Closest thing I can find in history is the founding of the precursor bank to BBVA, on May 28, 1857.

Reply:
I pulled it from a `timestamp` test that was failing for closer inspection.